src/services/functionAppService.ts

```diff
@@ -40,17 +41,10 @@ export class FunctionAppService extends BaseService {
 
   public async getMasterKey(functionApp?: Site) {
     functionApp = functionApp || await this.get();
-    const adminToken = await this.getAuthKey(functionApp);
-    const keyUrl = `https://${functionApp.defaultHostName}/admin/host/systemkeys/_master`;
+    const keyUrl = `${this.baseUrl}${functionApp.id}/host/default/listkeys?api-version=2016-08-01`;
```

Comment:
function app roll out new url?

Reply:
Not sure. This may have been an artifact of `0.7.0`. But @wbreza found this and updated it... Unfortunately I squashed his commits by accident 🤦‍♂️